### ISSUE TITLE
Screen Reader Friendly can be determined in two ways that should be described.

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#123]
AccessModeSufficent with a value of textual is one way of determining that a screen reader will be able to access all the information. Graphics will need alt text and/or extended descriptions for textual to be applied.

Also, if the publication conforms to WCAG at the A or AA level, all graphical content must have a text equivalent.  

Our guidelines should point out that the term screen reader friendly can use either or both of these metadata items.

**Comment by michael-n-cooper** [comment]
I removed the a11y-needs-resolution label as that triggers APA tracking of comments filed on W3C specifications. I think that is not the intent for issues in this repository.

**Comment by GeorgeKerscher** [comment]
We clearly address this in the techniques

**Comment by HadrienGardeur** [comment]
A number of metadata can be inferred from the fact that a publication conforms to WCAG A/AA.

We could either:

- mention conformance to WCAG as an alternate way of determining the presence of these features
- or address inferences from WCAG A/AA as its own section

Inferences are never without risk since a publisher/author could mistakenly indicate that a publication conforms to WCAG.

**Comment by rickj** [comment]
With the US legislation using WCAG 2.1 AA as the requirement, I would want more clarity in the techniques documents.  The inferences @HadrienGardeur references are critical for those who are not immersed in the details to understand what is being claimed.

**Comment by wareid** [comment]
Agreed on a crosswalk of WCAG inferences, but I do agree that if a publication is claiming WCAG 2.x AA conformance, screen reader friendly is a logical inference, especially considering how much of WCAG is devoted to ensuring good screen reader compatibility. 

One helpful breakdown for publishers might be what in WCAG applies to book content. WCAG is huge and covers a lot of use cases, but many of them don't apply to ebooks, such as the authentication requirements. 

We did a version of this kind of breakdown for the FXL accessibility doc, it wouldn't be too hard to extract and expand that content for reflow.

**Comment by clapierre** [comment]
> Also worth pointing out that with over 2 million EPUB files available, I couldn't find a single publication that claimed conformance to level A, every single one of them targeted AA.

We have had a few GCA publishers publish a few books at the WCAG 2.0-A level because they failed to meet color contrast requirements due to some unique requirements enforced by the author which could not be adjusted and also mentioned why they are only Single A in the accessibilitySummary.

Others did the same because there were too many language shifts throughout the book and was too costly to tag each of them correctly.

**Comment by rickj** [comment]
We have 34,228 titles with an explicit WCAG level called out in the conformance metadata.

Rick Johnson | Co-Founder and Vice President of Solutions Engineering and Accessibility
VitalSource Technologies, LLC
get.vitalsource.com <https://get.vitalsource.com/>



From: Charles LaPierre ***@***.***>
Date: Wednesday, September 11, 2024 at 10:50 AM
To: w3c/publ-a11y ***@***.***>
Cc: Johnson, Rick ***@***.***>, Comment ***@***.***>
Subject: Re: [w3c/publ-a11y] Screen Reader Friendly can be determined in two ways that should be described. (Issue #123)

Also worth pointing out that with over 2 million EPUB files available, I couldn't find a single publication that claimed conformance to level A, every single one of them targeted AA.

We have had a few GCA publishers publish a few books at the WCAG 2.0-A level because they failed to meet color contrast requirements due to some unique requirements enforced by the author which could not be adjusted and also mentioned why they are only Single A in the accessibilitySummary.

Others did the same because there were too many language shifts throughout the book and was too costly to tag each of them correctly.
